### ISSUE TITLE
Bug: `litestar run` CLI has several readability issues

### ISSUE THREAD

**Issue by sobolevn** [litestar-org/litestar#3822]
### Description

First problem: low contrast on light theme:

<img width="788" alt="Снимок экрана 2024-10-16 в 23 04 11" src="https://github.com/user-attachments/assets/7cb84b46-ea93-4b53-84d8-a7fd7f05d6f2">

I can hardly read what grey and yellow texts say.
One can argue that this is a problem of my setup / theme, but I've never seen this before in other apps.

Second problem:
<img width="788" alt="Снимок экрана 2024-10-16 в 23 04 06" src="https://github.com/user-attachments/assets/8af38089-6d61-43c9-bc73-c1e6ad189ba1">

Option's name of `--create-self-signed-c...` (certificate?) is cut. I think that this is the most important part of the help here. And it should not cut the options' names.

### Steps to reproduce

```bash
1. Run `litestar run -h`
```

### Litestar Version

main

### Platform

- [ ] Linux
- [ ] Mac
- [ ] Windows
- [ ] Other (Please specify in the description above)

**Comment by provinzkraut** [comment]
I don't think we can change this, as it's something coming from [rich-click](https://github.com/ewels/rich-click). That library has caused us a lot of trouble though, so maybe it's time to ditch this in 3.0 (or just move away from click in general :)).


@JacobCoffee, @DanCardin

**Comment by Alc-Alc** [comment]
@provinzkraut is there a reason max width is set to 80 here, setting that to a different value should help with part of this issue

https://github.com/litestar-org/litestar/blob/b79519d40eeff160905c60de3da25ac10a1e9fca/litestar/cli/__init__.py#L17

**Comment by provinzkraut** [comment]
> @provinzkraut is there a reason max width is set to 80 here, setting that to a different value should help with part of this issue
> 
> https://github.com/litestar-org/litestar/blob/b79519d40eeff160905c60de3da25ac10a1e9fca/litestar/cli/__init__.py#L17

Not that I know of. I think @cofin introduced this?
